Documentation/platforms.md

```diff
@@ -0,0 +1,85 @@
+# Target Platforms
+
+MonoGame supports the following systems:
+
+- Windows
+- Mac
+- Linux
+- Android
+- iOS
+- PlayStation 4
+- PlayStation Vita
+- Xbox One
+- Nintendo Switch
+- Google Stadia
+
+There are different implementations of MonoGame that we call target platforms (or just platforms).
+The platforms mostly correspond to the systems MonoGame supports but some platforms
+support multiple systems. Platforms for systems that require registration such as PlayStation
+and Xbox are not publicly available. To get access to those platforms, contact Tom Spilman
+at ???. MonoGame documentation for closed platforms is available in their respective repositories.
```

Comment:
@tomspilman What should I put here?

Reply:
The safest is to indicate users to contact their console account manager (which will verify their registration and loop back to Tom to confirm).